.travis.yml

```diff
@@ -146,4 +146,4 @@ jobs:
 branches:
   only:
     - master
-    - ^v\d+.*  # To build tag pushes for releases
+    - /^v\d+.*/  # To build tag pushes for releases
```

Comment:
I'm not sure if we care about the strictness of only matching valid release tags, but this regex would also match things like `v1111`. There's a pretty good regex in the [Travis documentation](https://docs.travis-ci.com/user/customizing-the-build/#safelisting-or-blocklisting-branches) for matching release versions:

`/^v\d+\.\d+(\.\d+)?(-\S*)?$/`

This would just be an optimization, so no need to block approval on this